src/main/java/com/zipcodewilmington/looplabs/StringDuplicateDeleter.java

```diff
@@ -5,4 +5,54 @@
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class StringDuplicateDeleter extends DuplicateDeleter<String> {
+
+    public StringDuplicateDeleter(String[] stringArray) {
+        super(stringArray);
+    }
+
+    @Override
+    public String[] removeDuplicatesExactly(int exactNumberOfDuplications) {
+        StringBuilder dupes = new StringBuilder();
+        int dupeCount = 0;
+        for (int i = 0; i < this.array.length; i++) {
+            for (int j = 0; j < this.array.length; j++) {
+                if (this.array[i].equals(this.array[j])) {
+                    dupeCount++;
+                }
+            }
+            if (!(dupeCount == exactNumberOfDuplications)) {
+                dupes.append(this.array[i]);
+                dupes.append(",");
+            }
+            dupeCount = 0;
+        }
+        String[] dupesRemoved = dupes.toString().split(",");
+        return dupesRemoved;
```

Comment:
dupes removed is a confusing name. I thought that it was the duplicates you had removed until I saw you were returning it. Consider a name like `arrayWithDuplicatesRemoved`.